shared/locales/de/website-videos.json

```diff
@@ -0,0 +1,80 @@
+{
+	"credits": "Credits",
+	"id": {
+		"video-01": "896169775",
+		"video-02": "488184818",
+		"video-03": "840215695"
+	},
+	"video-01": {
+		"credits": [
+			{
```

Comment:
If you added the credits here, it makes sense to remove them from the old location. Otherwise, we have the same information twice...

Reply:
@mkue Good point. I deleted the duplicated credits.